xml/System.Runtime.Serialization.Formatters.Binary/BinaryFormatter.xml

```diff
@@ -289,11 +292,15 @@
         <ReturnType>System.Runtime.Serialization.SerializationBinder</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets an object of type <see cref="T:System.Runtime.Serialization.SerializationBinder" /> that controls the binding of a serialized object to a type.</summary>
+        <summary>Insecure: Gets or sets an object of type <see cref="T:System.Runtime.Serialization.SerializationBinder" /> that controls the binding of a serialized object to a type.</summary>
```

Comment:
Are people going to understand why it says "insecure" there?

Reply:
How much can we dd to the ` <summary/>` ? Open to suggestions. I was hoping they'd see that and keep reading.